behave_test_automation/pah_behave/features/steps/confirm_drive_steps.py

```diff
@@ -0,0 +1,34 @@
+from behave import given, then, when, step
+from behave.matchers import use_step_matcher
+
+use_step_matcher("re")
+
+
+@given('User navigates to confirm drive')
+def navigate_to_confirm_drive(context):
+    context.confirm_drive_page.navigate_to_confirm_drive()
+
+
+@step('User submits "([^"]*)" driver code')
```

Comment:
I was hoping that you'd change the regexps to the parametrized placeholders elsewhere, as well ;P

Reply:
Shame on me, fixed :). I modified those which were possible to. When I use empty values in scenario outline, I have to support those steps via use_step_matcher, because otherwise, the step is not being matched with the snippet.

Reply:
oh, sorry, didn't knew that.